doc/windows/package-manager/package/winget-validation.md

```diff
@@ -53,7 +53,7 @@ immediately. Some may trigger manual validation.
 | **Label** | **Details** |
 |--------------|-------------|
 |<img width=200/>|<img width=500/>|
-| **Binary-Validation-Error** | The application included in this **Pull Request** failed to pass the **Installers Scan** test. This test is designed to ensure that the application installs on all environments without warningsFor further details on this error, see [binary validation errors](.\b. inary-validation-errors.md). |
+| **Binary-Validation-Error** | The application included in this **Pull Request** failed to pass the **Installers Scan** test. This test is designed to ensure that the application installs on all environments without warnings.  For further details on this error, see [binary validation errors](.\binary-validation-errors.md). |
```

Comment:
Btw, is there a specific reason to use `.\` instead of `./`? The former is fine on Windows, but doesn't work well anywhere else.

https://github.com/microsoft/winget-cli/blob/fed281ed9170c5902c5c4e7238415390d1c4e900/doc/windows/package-manager/package/.%5Cbinary-validation-errors.md